### PR TITLE
feat: Add support for Azure DevOps

### PR DESCRIPTION
**Pull request by tido64**

### CODE REVIEW

File: src/AzureDevOpsClient.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      rightFileEnd: { line },
+      rightFileStart: start_line ? { line: start_line } : { line },
+    },
+    pullRequestThreadContext: {
+      changeTrackingId: iteration,
+      iterationContext: {
+        firstComparingIteration: iteration,
+        secondComparingIteration: iteration,
+      },
+    },
```
Comment (bweggersen) [inline src/AzureDevOpsClient.js:57]: What's the significance with of this?

**Review by bweggersen — APPROVED** [review]
Great work 💯

File: src/AzureDevOpsClient.js (hunk 1 above)
Comment (tido64) [reply src/AzureDevOpsClient.js:57]: It tells ADO which iteration your comment was made on. If we don't provide it, ADO will say that it cannot find the relevant code lines or something to that effect. For most of our use cases, we want to make comments on the very latest iteration.

**Pull request merged**